proto/buf.yaml

```diff
@@ -1,6 +1,4 @@
 version: v1beta1
-deps:
-- buf.build/googleapis/googleapis
```

Comment:
:slightly_frowning_face: I think I needed to add this to get proto generation to work correctly on a branch

Reply:
That's weird: it broke suddenly in CI for me :/